client/app/router.js

```diff
@@ -9,4 +9,5 @@ export default class Router extends EmberRouter {
 
 Router.map(function() { // eslint-disable-line
   this.route('projects');
```

Comment:
wondering if we can nest authenticated routes under 
```
  this.route('authenticated', { path: '' }, function() {
```

Reply:
great idea. enhancement for later?